parm/atm/obs/config/sondes.yaml

```diff
@@ -1,74 +1,44 @@
 obs space:
   name: sondes
   obsdatain:
-    engine:
+    engine: 
       type: H5File
-      obsfile: $(OBS_DIR)/$(OBS_PREFIX)sondes.$(OBS_DATE).nc4
-    obsgrouping:
-      group variables: ["station_id"]
-      sort variable: "air_pressure"
-      sort order: "descending"
+      obsfile: sondes_obs_$(OBS_DATE).nc4
   obsdataout:
     engine:
       type: H5File
-      obsfile: $(DIAG_DIR)/diag_sondes_$(OBS_DATE).nc4
-  io pool:
-    max pool size: 1
-  simulated variables: [eastward_wind, northward_wind, air_temperature, specific_humidity, surface_pressure]
-get values:
-  interpolation method: $(INTERP_METHOD)
+      obsfile: sondes_diag_$(OBS_DATE).nc4
+  simulated variables: [surface_pressure, air_temperature, eastward_wind, northward_wind, specific_humidity]
+geovals:
```

Comment:
geovals, vector ref, tolerance do not go in the `config` YAMLs. Also the `io pool` section needs to stay

Reply:
Updated for the required changes.